docs/upgrade.html

```diff
@@ -19,6 +19,17 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h5><a id="upgrade_241_notable" href="#upgrade_241_notable">Notable changes in 2.4.1</a></h5>
```

Comment:
I think this needs to go down below the `upgrade_2_4_0` section.  See, for example, ``upgrade_221_notable` and `upgrade_111_notable` sections, which appear below the `upgrade_2_2_0` and `upgrade_1_1_0` sections, respectively.

Reply:
Ack.  That's where I had it initially, but I decided to put it up on top.  I'll revert and move it back to where it should be.  Thanks for the review!